docs/tutorial/04_integrating_magma_access_gateway_with_magma_orchestrator.md

```diff
@@ -69,18 +69,17 @@ unit-magma-access-gateway-operator-0:
 ### Create a network in the `magma-test` organization:
 
 1. Login to the `magma-test` organization at this address: `https://magma-test.nms.<your domain>`. Use the credentials from the previous step.
-2. On the left pane, click on "Networks"
-3. Click on "Add Network"
-4. Fill in the following values:
+2. Click on "Add Network"
+3. Fill in the following values:
     * Network ID: `my-network`
     * Network Name: `my-network`
     * Description: `my-network`
     * Network Type: `lte`
-5. Refresh the page. You should now see your network dashboard
```

Comment:
I do remember the need for refreshing. IDK if that behavior has changed.

Reply:
It may have been necessary with 1.6, but on the 1.8 I just installed it refreshed automatically.